src/Commands/FrequencySetCommand.php

```diff
@@ -2,80 +2,65 @@
 
 namespace Pantheon\TerminusAutopilot\Commands;
 
+use Pantheon\Terminus\Commands\TerminusCommand;
 use Pantheon\Terminus\Request\RequestAwareInterface;
-use Pantheon\Terminus\Request\RequestAwareTrait;
+use Pantheon\Terminus\Site\SiteAwareInterface;
 use Pantheon\Terminus\Site\SiteAwareTrait;
-use Pantheon\TerminusAutopilot\Autopilot\AutopilotAwareTrait;
+use Pantheon\TerminusAutopilot\AutopilotApi\AutopilotClientAwareTrait;
 
 /**
- * Autopilot status check.
+ * Class FrequencySetCommand.
  */
-class FrequencySetCommand extends AutopilotCommandBase implements RequestAwareInterface
+class FrequencySetCommand extends TerminusCommand implements RequestAwareInterface, SiteAwareInterface
 {
-
-    use AutopilotAwareTrait;
-    use RequestAwareTrait;
+    use AutopilotClientAwareTrait;
     use SiteAwareTrait;
 
     /**
-     * Set Autopilot run frequency for a specific site.
+     * Get or set Autopilot run frequency.
      *
+     * @command site:autopilot:frequency
+     * @aliases ap-frequency
      * @authorize
      * @filter-output
      *
-     * @command site:autopilot:frequency
-     * @aliases ap-frequency-set
+     * @usage <site_id> Get Autopilot run frequency
+     * @usage <site_id> <frequency> Set Autopilot run frequency
      *
-     * @param string $site_id Long form site ID.
-     * @param string $frequency Frequency for Terminus to run.
-     *  Available options: MANUAL, MONTHLY, WEEKLY, DAILY.
-     * @param array $options
+     * @param string $site_id Site name
+     * @param string|null $frequency Autopilot run frequency.
+     *   Available options: manual, monthly, weekly, daily.
      *
-     * @return bool
-     * @throws \Pantheon\Terminus\Exceptions\TerminusException|\GuzzleHttp\Exception\GuzzleException
+     * @return string|null
+     *
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     * @throws \Psr\Container\ContainerExceptionInterface
+     * @throws \Psr\Container\NotFoundExceptionInterface
      */
-    public function autopilotFrequencySet(
-        string $site_id,
-        string $frequency,
-        array $options = ['debug' => false]
-    ): bool {
-
-        // Allow frequency to be case-insensitive for user friendliness.
-        $frequency = strtoupper($frequency);
+    public function autopilotFrequencySet(string $site_id, string $frequency = null): ?string
```

Comment:
You're now using the same command/function to set and get. Could you please rename the function for clarity?